jenkins/kola/packet.sh

```diff
@@ -48,6 +48,7 @@ cl_internet_included="$(set -o noglob; bin/kola list --platform=packet --filter
 if [[ "${BOARD}" == "amd64-usr" ]] && [[ "${cl_internet_included}" != ""  ]]; then
   for INSTANCE in c3.medium.x86 m3.large.x86 s3.xlarge.x86 n2.xlarge.x86; do
     (
+    set +x
```

Comment:
This is probably a debugging leftover.

Reply:
No, it's the second commit to disable the printing of the OUTPUT="Long output with newlines" statement

Reply:
Oh indeed. I didn't check the commits.